docs/data/joy/components/drawer/drawer.md

```diff
@@ -7,64 +7,66 @@ githubLabel: 'component: drawer'
 
 # Drawer
 
-<p class="description">Navigation drawers provide quick access to other destinations in your app without removing the user out of context.</p>
+<p class="description">Navigation drawers provide quick access to other app areas without removing the user out of context.</p>
 
 ## Introduction
 
+The Drawer component is commonly used as a type of menu on desktop and as a dialog on mobile (similar to [Apple's sheets](https://developer.apple.com/design/human-interface-guidelines/sheets)).
+
 {{"demo": "DrawerUsage.js", "hideToolbar": true, "bg": "gradient"}}
 
 ## Basic
 
-The navigation drawers can toggle open or closed. Closed by default, the drawer opens temporarily above all other content until a section is selected.
-
-The Drawer can be cancelled by clicking the overlay or pressing the Esc key.
-It closes when an item is selected, handled by controlling the `open` prop.
+The Drawer can be closed by clicking on the overlay or pressing the <kbd>Esc</kbd> key.
+It will also close if an item inside of it is selected, which can be controlled by the `open` prop.
 
 {{"demo": "DrawerBasic.js"}}
 
 ## Customization
 
 ### Anchor
 
-You can use the `anchor` prop for specifying where the drawer should appear from.
+Use the anchor prop to specify where the drawer should appear from.
 
 {{"demo": "DrawerAnchor.js"}}
 
 ### Close button
 
-Use the ModalClose component to add a close button to the drawer.
+Use the Modal Close component to add a close button to the Drawer.
 
 {{"demo": "DrawerCloseButton.js"}}
 
 ### Size
 
-The `size` prop allows you to adjust the size of the drawer.
+Use the `size` prop to adjust the Drawer's width.
```

Comment:
@siriwatknp Is it accurate to say that this prop only controls the _width_? Because when it's anchored to the top or bottom, it should instead control the _height,_ right?

Reply:
Yes, it controls the `width` or `height` based on the anchor.